masu/processor/tasks.py

```diff
@@ -0,0 +1,69 @@
+#
+# Copyright 2018 Red Hat, Inc.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+#
+"""Celery task definitions."""
+
+# pylint: disable=unused-argument, fixme
+# FIXME: temporary module-wide disable until tasks are fully implemented.
+
+from unittest import mock
+
+from celery.utils.log import get_task_logger
+
+from .celery.worker import QUEUE
+from .exceptions import MasuProcessingError
+
+
+LOG = get_task_logger(__name__)
+
+
+@QUEUE.task(autoretry_for=(MasuProcessingError,),
```

Comment:
Will this QUEUE decorator cause make_celery to be ran each time?  It's been awhile since I've used celery but I remember there being just one instance of Celery() something like app = Celery() and you use @app.task.